packages/loader/container-loader/src/loader.ts

```diff
@@ -428,3 +431,59 @@ export class Loader implements IHostLoader {
 		);
 	}
 }
+
+/**
+ * {@inheritDoc @fluidframework/container-definitions#ILoader.resolve}
+ * @legacy
+ * @alpha
+ */
+export async function resolve(
+	request: IRequest,
+	loaderProps: ILoaderProps,
+	pendingLocalState?: string,
+): Promise<IContainer> {
+	const loader = new Loader(loaderProps);
```

Comment:
could be done here, or as a follow up, but we should move the implementations to these functions, and have the loader use the functions

Reply:
But then who will create the loader object? @anthony-murphy